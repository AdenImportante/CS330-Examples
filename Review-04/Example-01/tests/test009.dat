#! /bin/sh

# Repeat test008 with attention to formatting

# Consider all whitespace and letter-case (upper/lower)
./storage test008.in 4 