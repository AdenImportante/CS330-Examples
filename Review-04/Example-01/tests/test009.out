Processing Log:
 (S) Hat
 (S) Boots

Player Storage Summary:
 -Used  50% of 4 slots
  Nme: Hat
  Dur: 1
  Def: 5
  Mtl: Gold
  Mdr: Protection (Lvl 3)
  Emt: water

  Nme: Boots
  Dur: 2
  Def: 4
  Mtl: Diamond
  Mdr: FeatherFalling (Lvl 4)
  Emt: fire

